### ISSUE TITLE
Issue with closure use on updateMarkup attribute

### ISSUE THREAD

**Issue by deuxni** [kartik-v/yii2-detail-view#122]
Closure cannot be used because the first parameter ($form) is always null : a first normalization of attributes is done with call of parseAttributeItem() and parseAtributeProp() inside, but $this->_form is  null during this step and the closure is called with that null value. To make this working, I have updated the parseAttributeProp() to call the closure if $this->_form is defined, so the closure is correctly called later into renderFormAttribute(): 
```php
protected function parseAttributeProp($setting)
    {
        /**
         * @var Closure|mixed $setting
         */
        return ($setting instanceof Closure && $this->_form) ? $setting($this->_form, $this) : $setting;
    }

**Comment by dmitriyha** [comment]
fixed in #120 ... now only if a merge would happen O.O

**Comment by kartik-v** [comment]
Closed via #120.
